pom.xml

```diff
@@ -298,6 +298,32 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>jacoco-report</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.8.5</version>
```

Comment:
versions should be managed via `pluginManagement` of `maven-parent`.
I created #83 for this.